src/cjs-download/index.ts

```diff
@@ -4,13 +4,27 @@ import consistentSort from "../lib/consistentSort"
 import convertOds from "./convertOds"
 import downloadCjsFile from "./downloadCjsFile"
 import getDownloadUrl from "./getDownloadUrl"
+import mapOffenceCodeData from "../map-data/mapOffenceCodeData"
+
+type OffenceCodeRecords = {
+  cjsCode: string
+  description?: string
+  homeOfficeClassification?: string | null
+  notifiableToHo?: string
+  recordCreated?: number[]
+  source?: string
+  offenceCategory?: string
+  offenceTitle?: string
+  recordableOnPnc?: string
+  resultHalfLifeHours?: string | null
+}
 
 export default async () => {
   console.log("Downloading CJS data")
   const downloadLocation = await getDownloadUrl()
   const fileContents = await downloadCjsFile(downloadLocation)
   const offenceCodes = convertOds(fileContents)
-  const data = consistentSort(offenceCodes)
+  const data = mapOffenceCodeData(consistentSort(offenceCodes) as OffenceCodeRecords[])
```

Comment:
Might be better to just convert the relevant fields to boolean in the earlier step that ouputs the data rather than adding a second layer of processing here